src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -188,6 +188,16 @@
   private WebElement footer;
   @FindBy(css = "#globalNavigation")
   private WebElement globalNavigationBar;
+  @FindBy(css = "#WikiaPageHeader > nav.wikia-menu-button > span.drop")
+  private WebElement getEditButton;
+  @FindBy(css = "#ca-flags")
+  private WebElement flagButton;
+  @FindBy(css = "#flagsEditForm ul input[type=checkbox]")
+  private List<WebElement> allFlags;
```

Comment:
You are adding this variable to WikiBasePageObject. I went to an article tried this selector and found no elements. This means that this element is not wide enough for setting it to WikiBasePageObject. Please find or create more specific PageObject or ComponentObject